docs/Intents.md

```diff
@@ -0,0 +1,56 @@
+# NOTICE - This document is not yet up to date
```

Comment:
I am gonna wait with refactoring Intents.md until we are sure it does not change again.